extensions/default/src/ViewerLayout/index.jsx

```diff
@@ -55,16 +55,22 @@ function ViewerLayout({
     };
   };
 
+  const defaultTool = { icon: 'tool-more-menu', label: 'More' };
   const [toolbars, setToolbars] = useState({ primary: [], secondary: [] });
+  const [activeTool, setActiveTool] = useState(defaultTool);
+  const onSecondaryClickHandler = () => setActiveTool(defaultTool);
```

Comment:
The secondary toolbar shouldn't impact which icon is shown for the nested/more menu?
Only buttons _in_ the nested menu that are tools that can be set?

Reply:
I did this to get the nested menu back to its original icon, I'll take a better look and understand the existent code, I'm missing a good part of it. Thanks.